tdrs-frontend/src/reducers/reports.js

```diff
@@ -0,0 +1,19 @@
+import { SET_YEAR } from '../actions/reports'
+
+const initialState = {
+  year: 2020,
+}
+
+const reports = (state = initialState, action) => {
```

Comment:
Creates the `reports` reducer that will be expanded in the next PR's but currently sets the year under the reports module.